pandas/core/frame.py

```diff
@@ -2931,7 +2950,7 @@ def T(self) -> DataFrame:
     # ----------------------------------------------------------------------
     # Indexing Methods
 
-    def _ixs(self, i: int, axis: int = 0):
+    def _ixs(self, i: int, axis: Axis = 0):
```

Comment:
`Axis` includes `str`, we can only use the `Axis` alias if we use `axis = self._get_axis_number(axis)` or pass it onto a method that does so. Here we don't do that lookup so should use `int`

Reply:
Right, fixed